nucypher-core/src/revocation_order.rs

```diff
@@ -13,7 +13,8 @@ use crate::versioning::{
 /// Represents a string used by characters to perform a revocation on a specific Ursula.
 #[derive(PartialEq, Debug, Serialize, Deserialize)]
 pub struct RevocationOrder {
-    staker_address: Address,
+    /// The address of the Ursula that is being revoked.
+    pub staker_address: Address,
```

Comment:
I guess we're finally doing something with revocation orders? Is it used anywhere yet, or is it just a preparation?

Also if it's exposed in JS, it needs to be exposed in Python bindings as well.

Reply:
This is probably a bit premature given that IDK how revocations are going to work post-v6.0.0